packages/odyssey-storybook/src/components/odyssey-mui/SearchField/SearchField.stories.tsx

```diff
@@ -169,8 +169,17 @@ export const ControlledSearch: StoryObj<typeof SearchField> = {
         setControlledValue(event.target.value),
       [],
     );
+    const onClear = () => {
+      setControlledValue("");
+    };
+
     return (
-      <SearchField {...props} value={constrolledValue} onChange={onChange} />
+      <SearchField
+        {...props}
+        value={constrolledValue}
+        onChange={onChange}
+        onClear={onClear}
```

Comment:
I wonder if we wanna expose this prop at the component level.

Reply:
It is exposed on the component. Just didn't exist in the story. We were rendering a non-functioning `x` icon in the story